scss/plugins/angular-ui-bootstrap-reskin/angular-ui-bootstrap-reskin.scss

```diff
@@ -35,3 +35,5 @@ $vars: $pluginVars;
 @import "components/collapse";
 // Daterange
 @import "components/daterange";
+// Day Block
+@import "components/day-block";
```

Comment:
Please, can you tell me where I should make the import? Is lucca-ui.namespaced.scss a better place?